package.json

```
@@ -1,29 +1,53 @@
 {
-  "name": "@bbc/digital-paper-edit-audio-converter",
+  "name": "digital-paper-edit-audio-converter",
   "version": "1.0.0",
-  "description": "",
+  "description": "Service to standardise audio input for DPE Speech-To-Text transcription",
   "main": "index.js",
   "scripts": {
-    "start": "nodemon server.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "node server.js",
+    "test": "mocha",
+    "lint": "eslint .",
+    "spec": "speculate --release ${BUILD_NUMBER}"
   },
-  "author": "",
-  "contributors": [
-    ""
-  ],
-  "bugs": {
-    "url": "https://github.com/bbc/digital-paper-edit-audio-converter/issues"
-  },
-  "homepage": "https://github.com/bbc/digital-paper-edit-audio-converter",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/bbc/digital-paper-edit-audio-converter.git"
+    "url": "https://github.com/https://github.com/bbc/digital-paper-edit-audio-converter.git"
```

There seems to be a repetition here? `https://github.com/https://github.com/` should be like below?
```suggestion
    "url": "https://github.com/bbc/digital-paper-edit-audio-converter.git"
```

---

Ooh yeah good catch. This has happened because maybe I put in the full github URL during the generator CLI question - it should just be the repo-name.